Brokerages/GDAX/GDAXDataQueueHandler.cs

```diff
@@ -35,6 +36,12 @@ public class GDAXDataQueueHandler : GDAXBrokerage, IDataQueueHandler
             IPriceProvider priceProvider, IDataAggregator aggregator)
             : base(wssUrl, websocket, restClient, apiKey, apiSecret, passPhrase, algorithm, priceProvider, aggregator)
         {
+            var subscriptionManager = new EventBasedDataQueueHandlerSubscriptionManager();
```

Comment:
Think here we have base `protected DataQueueHandlerSubscriptionManager _subscriptionManager` from `BaseWebsocketsBrokerage` ?

Reply:
yeah. exactly. and it's assigned right there :)

Reply:
oh 😅 I see it now, confusing, same thought for `PolygonDataQueueHandler`. I'd suggest directly setting the final property from the start, but it's okay like this too